src/test/java/com/wikia/webdriver/testcases/discussions/FollowingPostTests.java

```diff
@@ -211,9 +213,10 @@ public void userOnDesktopCanUnfollowPostOnUserPostsPage() {
   /**
    * By default all posts on "Followed" tab are followed.
    */
-  @Test(groups = "discussions-userDesktopFollowingPost")
+  @Test(groups = "discussions-userDesktopFollowingPost", enabled = false)
```

Comment:
can you please add "TODO" in a comment with information when to enable these tests again?